packages/blade/src/components/Dropdown/Dropdown.tsx

```diff
@@ -111,11 +120,16 @@ const _Dropdown = ({
       hasLabelOnLeft,
       setHasLabelOnLeft,
       dropdownTriggerer: dropdownTriggerer.current,
+      changeCallbackTriggerer,
+      setChangeCallbackTriggerer,
+      isControlled,
+      setIsControlled,
     }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why do we need to disable this eslint warning or why do we not put all the deps in the dependency array?

Reply:
we haven't added the setter functions of state to dependency array. We don't need them no? in most cases adding something that is not needed in dependency array has ended me with infinite loops or some rendering issues lol so I prefer to disable this as long as some prop is not consciously needed.